.circleci/config.yml

```diff
@@ -36,40 +15,28 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v3-rbenv-dependencies-20230418-{{ checksum ".ruby-version" }}
-          - v3-dependencies-v3-{{ checksum "Gemfile.lock" }}
-
-      - run:
-          name: rbenv update
-          command: |
-            rbenv_update_env
-
-      - save_cache:
-          paths:
-            - ~/.rbenv
-          key: v3-rbenv-dependencies-20230418-{{ checksum ".ruby-version" }}
+          - bundle-{{ checksum "Gemfile.lock" }}
 
       - run:
           name: bundle install
           command: |
             # required for the license_finder
-            # The Gemfile.lock specifies nokogiri (1.14.3)
+            # The Gemfile.lock specifies nokogiri (1.16.2)
             # but bundler would install native gem instead if not forced
             bundle config set --local force_ruby_platform true
-            bundle_install_rubies
+            bundle config set --local path ./vendor/bundle
```

Comment:
maybe better set deployment mode like in porta?

Reply:
What's that?

Reply:
```
bundle config set --local deployment 'true'
```

https://bundler.io/guides/deploying.html#deploying-your-application

I think this suggestion is good actually.

Reply:
It looks good, maybe we can remove the above lines about nokogiri. Let me try

Reply:
Done. I think we can't remove the `force_ruby_platform` though.